packages/nocodb/docker/start-litestream.sh

```diff
@@ -1,26 +1,37 @@
 #!/bin/sh
 
-#sleep 5
-
-if [ -n "${NC_TOOL_DIR}" ]; then
+if [ ! -d "${NC_TOOL_DIR}" ] ; then
   mkdir -p "$NC_TOOL_DIR"
 fi
 
-if [ -n "${AWS_ACCESS_KEY_ID}" ] && [ -n "${AWS_SECRET_ACCESS_KEY}" ] && [ -n "${AWS_BUCKET}" ] && [ -n "${AWS_BUCKET_PATH}" ]; then
+use_litestream() {
+     [ -z "${NC_DB}" ] \
+  && [ -z "${NC_DB_JSON}" ] \
+  && [ -z "${NC_DB_JSON_FILE}" ] \
+  && [ -z "${DATABASE_URL}" ] \
+  && [ -z "${DATABASE_URL_FILE}" ] \
+  && [ -z "${NC_MINIMAL_DBS}" ] \
+  && [ -n "${LITESTREAM_S3_ENDPOINT}" ] \
+  && [ -n "${LITESTREAM_S3_BUCKET}" ] \
+  && [ -n "${LITESTREAM_ACCESS_KEY_ID}" ] \
+  && [ -n "${LITESTREAM_SECRET_ACCESS_KEY}" ]
+}
+
+if use_litestream ; then
 
-  if [ -f "${NC_TOOL_DIR}noco.db" ]
-  then
+  if [ -f "${NC_TOOL_DIR}noco.db" ] ; then
     rm "${NC_TOOL_DIR}noco.db"
-    rm "${NC_TOOL_DIR}noco.db-shm"
-    rm "${NC_TOOL_DIR}noco.db-wal"
+    rm -f "${NC_TOOL_DIR}noco.db-shm"
+    rm -f "${NC_TOOL_DIR}noco.db-wal"
   fi
 
-  /usr/src/appEntry/litestream restore -o "${NC_TOOL_DIR}noco.db" "s3://$AWS_BUCKET/$AWS_BUCKET_PATH"
-  if [ ! -f "${NC_TOOL_DIR}noco.db" ]
-  then
+  litestream restore "${NC_TOOL_DIR}noco.db"
+
+  if [ ! -f "${NC_TOOL_DIR}noco.db" ] ; then
     touch "${NC_TOOL_DIR}noco.db"
   fi
-  /usr/src/appEntry/litestream replicate "${NC_TOOL_DIR}noco.db" "s3://$AWS_BUCKET/$AWS_BUCKET_PATH" &
+
+  litestream replicate &
```

Comment:
I think here Ffull path is required since default working directory is `/usr/src/app/` and `litestream` is within `/usr/src/appEntry/`.

Reply:
Nope, the litestream binary is at `/usr/local/bin/litestream` now. In Alpine Linux as well as other standard-conforming Linux distros, `/usr/local/bin` is the [most suitable place to store this kind of binary](https://unix.stackexchange.com/a/4187/201803). It is on `PATH` by default, so we can simply call `litestream`. ☺️

Reply:
Sorry I missed that part, looks good then.